docs/ref/models/querysets.txt

```diff
@@ -2246,6 +2252,9 @@ them, but it has a few caveats:
   extra query per ancestor.
 * When an individual batch contains duplicates, only the first instance in that
   batch will result in an update.
+* The return value of updated objects is expected to be same as the number of 
+  objects passed. Although, it may be lesser in case there are duplicate objects
+  passed in the same batch.
```

Comment:
There are also different reasons like race-conditions. IMO in this case less is more, I would remove this _"caveat"_ it can be confusing.
```suggestion
```

Reply:
I agree with your point about the race condition. This caveat was added as per the https://github.com/django/django/pull/14125#pullrequestreview-611738108. Do you still want me to remove it?

Reply:
We already have a note about duplicates, IMO it's unnecessary and add another one.

@pope1ni What do you think?

Reply:
i have now used the wording as suggested by Nick. Please let me know if this needs to removed or changed.

Reply:
question: shouldn't we give a link to something like a wikipedia article or just assume that the readers will be familiar with the context of the word `race` here.